ui/app/rif/pages/domainsPage/domainsPage.js

```diff
@@ -40,27 +40,39 @@ class DomainsScreen extends Component {
         }} id="chipletTitle" className={'chiplet-title'}>
           {data.name}
         </div>
-        <div id="chipletDescription" className={'chiplet-description'}>
-          <div id="chipletExpiration">
-            <span>Expires on: {data.details ? data.details.expiration : 'n/a'}</span>
-          </div>
-          <div id="chipletRenew">
-            <span>Auto-renew: <a href={data.details ? this.props.setAutoRenew() : () => {}}>{data.details ? (data.details.autoRenew ? 'on' : 'off') : 'n/a'}</a></span>
-          </div>
-        </div>
+        {/*<div id="chipletDescription" className={'chiplet-description'}>*/}
```

Comment:
I think before merging this PR we should delete this comments

Reply:
I think is just a change on the designs, that's why he commented that out, maybe we can remove that but i don't know

Reply:
After talking with @anarancio,  we conclude that we should keep that for now.

Reply:
Done